### DISCUSSION TITLE
[Remix] Error: ENOENT: no such file or directory, copyfile '/var/task/node_modules/@vercel/remix/default-server.js'

### DISCUSSION THREAD

**Discussion by danthareja** [vercel/community#1282] [Help] [answered]
### Summary

My remix deployment was failing after build:

```
Error: ENOENT: no such file or directory, copyfile '/var/task/node_modules/@vercel/remix/default-server.js' -> '/vercel/path0/build/__vc_handler.js'
```

### Additional information

I was able to reproduce this by deploying the [remix-boilerplate](https://vercel.com/templates/remix/remix-boilerplate), so I believe it to be an upstream issue.

### Example

![New_Project_–_Vercel](https://user-images.githubusercontent.com/6980359/213038039-f158e4ee-bb54-4da6-9fe5-720609e3d291.png)

**Comment by jezzzm** [comment]
Started getting this within the last 12 hours as well.

Can see that upstream `@vercel/node` and `@vercel/node-bridge` both had patches 4 hours ago.

Tried pinning those deps to earlier versions but problem still persists.

**Comment by InceptionCode** [comment]
Having this same issue.

**Reply by johannchen2021** [reply]
same issue with remix v1.10 as well

**Comment by leerob** [comment]
Thanks folks, we're looking into this!

**Reply by mcansh** [reply]
explicitly setting `serverBuildDirectory: "build"` made it work for my personal site (https://github.com/mcansh/mcan.sh), but `serverBuildTarget: "vercel"` sets `serverBuildPath` to `api/index.js` which ya'll check for and use the dir when copying the default server

**Reply by jezzzm** [reply]
> explicitly setting `serverBuildDirectory: "build"` made it work for my personal site (https://github.com/mcansh/mcan.sh), but `serverBuildTarget: "vercel"` sets `serverBuildPath` to `api/index.js` which ya'll check for and use the dir when copying the default server

legend, this worked for me as well!

**Comment by davisgreg1** [comment]
<img width="1200" alt="Screenshot 2023-01-18 at 12 51 47 AM" src="https://user-images.githubusercontent.com/22870001/213095068-47db95f0-3b9f-41cb-95a8-1313d87406b1.png">

Ditto... gonna be a long night.

**Comment by swarnava** [comment]
Hi,

Could you set `VERCEL_CLI_VERSION=vercel@28.11.1` in environment variable and let us know how it goes?

**PLEASE NOTE THAT THIS IS A TEMPORARY WORKAROUND AND YOU NEED TO REMOVE THIS ENVIRONMENT VARIABLE ONCE WE PUSH A FIX FOR THIS**

**Reply by ehwarren** [reply]
Works over here.

**Reply by swarnava** [reply]
Thanks for the confirmation. We will update this thread once our engineers push a fix for this.

**Reply by davisgreg1** [reply]
LGTM!

**Reply by gijsroge** [reply]
Thanks for the temp solution! Fixed the issue for me as well :)

**Reply by franklinjavier** [reply]
It worked here too, but it took 1m30s, the previous one is around 30s.

**Reply by fvaldes33** [reply]
Thank you for the quick temp solution

**Reply by Gaya** [reply]
I did the exact same thing in our Github Action and fixed the vercel package to a version where it was still working. Cheers

**Reply by danthareja** [reply]
This is a really handy workaround, thanks @swarnava!

**Reply by swarnava** [reply]
We have released a new version of Vercel CLI which should fix the problem. You can remove the Environment variabie and give it a try.

**Comment by swarnava** [comment, marked as answer]
We have released a new version of Vercel CLI which should fix the problem. You can remove the Environment variabie and give it a try.

**Reply by mmachatschek** [reply]
Hi @swarnava, we updated to the newest version ("removed the env var") and we started to receive this error message instead: https://github.com/orgs/vercel/discussions/1290

the error message from this discussion is gone

**Reply by swarnava** [reply]
This is a different error we are investigating currently. We will update shortly.

**Reply by swarnava** [reply]
We have pushed a fix for this ~30 mins ago. This should be resolved now.
